src/index.html

```diff
@@ -5,8 +5,262 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+          rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header-container">
+        <a href="#home" class="moyo-logo"></a>
+        <nav class="nav">
+          <ul class="nav__items">
+            <li>
+              <a href="#apple" class="nav__link is-active">apple</a>
+            </li>
+            <li><a href="#samsung" class="nav__link">samsung</a></li>
+            <li><a href="#smartphones" class="nav__link">smartphones</a></li>
+            <li>
+              <a href="#laptops-and-computers" class="nav__link"
+            data-qa="nav-hover">laptops & computers</a>
```

Comment:
```suggestion
              <a 
                href="#laptops-and-computers" 
                class="nav__link"
                data-qa="nav-hover"
              >
                laptops & computers
              </a>
```
Fix formatting everywhere